### DISCUSSION TITLE
Windows 10 or 11

### DISCUSSION THREAD

**Discussion by ghost** [beerisgood/Windows11_Hardening#2] [Q&A] [answered]
Would it be a bad idea from privacy and security preceptive to upgrade to Windows 11 from Windows 10?

**Comment by beerisgood** [comment, marked as answer]
Windows 11 is the better OS for both security and privacy.

**Reply by ghost** [reply]
Even despite the fact you need a Microsoft account?

**Reply by beerisgood** [reply]
Yes. You can config telemetry if this is what you’re worried about.

**Comment by beerisgood** [comment]
Did you have any further questions in that topic? Else i close it for now.
